### ISSUE TITLE
Curl cant resolve host

### ISSUE THREAD

**Issue by SebastianSikora** [itzg/dockerfiles#226]
Hello,
I am trying to use your config file to deploy vannila minecraft server, everything seems ok, but server cant download minecraft_server..jar so it cant boot.
```
Checking version information.
curl: (6) Could not resolve host: launchermeta.mojang.com
Checking type information.
Downloading minecraft_server..jar ...
curl: no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information
curl: no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information
server.properties already created, skipping
Skipping JSON check. No files present.
Setting initial memory to 1G and max to 1G
Error: Unable to access jarfile minecraft_server..jar
2018/07/09 15:43:33 Done

```

**Comment by itzg** [comment]
In cases like this it’s actually caused by name resolution outside of the container. Are you able to ping `launchermeta.mojang.com` from the host running the docker daemon?

**Comment by SebastianSikora** [comment]
Yes, i am able to ping.

**Comment by SebastianSikora** [comment]
I tried to wget from containter, but server cant resolve. If i do it outside i am able to resolve.

**Comment by SebastianSikora** [comment]
Problem fixed, i had bad --pod-network-cidr=10.244.0.0/16 while kubeadm init
